### PR TITLE
k8s.io/kube-openapi:upgrade k8s.io/klog to v2

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/kubernetes/kubernetes/issues/89961

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,7 +27,8 @@ require (
 	golang.org/x/tools v0.0.0-20181011042414-1f849cf54d09 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 	k8s.io/gengo v0.0.0-20190128074634-0689ccc1d7d6
-	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92
+	k8s.io/klog v1.0.0 // indirect
```
Comment (tanjunchen) [inline go.mod:30]: how to remove v1? 
it will reproduce again after running go mod tidy.

Comment (apelisse) [reply go.mod:30]: I had to install `github.com/KyleBanks/depth/cmd/depth` but this seems to work:
```
> for dir in $(go list ./...); do depth -explain k8s.io/klog "$dir"; done
k8s.io/kube-openapi/cmd/openapi-gen -> k8s.io/kube-openapi/cmd/openapi-gen/args -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/gengo/parser -> k8s.io/klog
k8s.io/kube-openapi/cmd/openapi-gen -> k8s.io/kube-openapi/cmd/openapi-gen/args -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/klog
k8s.io/kube-openapi/cmd/openapi-gen/args -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/gengo/parser -> k8s.io/klog
k8s.io/kube-openapi/cmd/openapi-gen/args -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/klog
k8s.io/kube-openapi/pkg/generators -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/gengo/parser -> k8s.io/klog
k8s.io/kube-openapi/pkg/generators -> k8s.io/gengo/args -> k8s.io/gengo/generator -> k8s.io/klog
```

File: go.mod [hunk 2]
```diff
@@ -27,7 +27,8 @@ require (
...
+	k8s.io/klog/v2 v2.0.0-rc.1
```
Comment (serathius) [inline go.mod:31]: Please change to v2.0.0

Comment (tanjunchen) [reply go.mod:31]: ok

File: go.mod (hunk 1 above)
Comment (dims) [reply go.mod:30]: @tanjunchen @apelisse i filed https://github.com/kubernetes/kube-openapi/pull/196 with this commit and another commit pulling in newer gengo which uses klog v2.

Comment (tanjunchen) [reply go.mod:30]: sure . Thanks

**Pull request merged**